spec_legacy/functional/projects_controller_spec.rb

```diff
@@ -64,14 +64,6 @@
     assert_select('a', { content: /Private child of eCookbook/ }, false)
   end
 
-  it 'should index atom' do
```

Comment:
Do I understand it right, that ```/projects.atom``` will not exists in future? If so: how do the links that are stored in peoples feed readers and systems deprecate? I am asking to make sure that everybody is clear on the consequences.